packages/app/pages/schedule.tsx

```diff
@@ -19,15 +30,15 @@ export default function Schedule() {
 
   const myEvents =
     data?.officeHours.map((e) => ({
-      title: e.title,
       start: e.startTime,
       end: e.endTime,
     })) ?? [];
 
   return (
-    <div style={{ height: "400px" }}>
-      <h1>{data?.name ?? ""}</h1>
-      <Calendar localizer={momentLocalizer(moment)} events={myEvents} />
-    </div>
+    <ScheduleCalendar
+      localizer={momentLocalizer(moment)}
+      events={myEvents}
+      defaultView={viewType}
+    />
```

Comment:
I found a solution for the month view, turns out your calendar needs to have a set height in order for it to display properly. I just pushed a commit with my change for this.

Regarding responsiveness, it seems like the crunkiness is coming from the Toolbar. It's probably worth looking into a custom Toolbar if you can't find a built in responsive solution in the library: https://jquense.github.io/react-big-calendar/examples/index.html#prop-components. This seems like a good example of a custom Toolbar: jquense/react-big-calendar#191.